CHANGELOG.md

```diff
@@ -71,6 +71,8 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/).
 - **CUMULUS-3876**
   - Fixed `s3-replicator` lambda cross region write failure
   - Added `target_region` variable to `tf-modules/s3-replicator` module
+- **CUMULUS-3981**
+  - Added required $metadata field when creating new instance of ServiceException.
```

Comment:
Is there AWS documentation that spells out why this is necessary? Very strange/concerning that this one error requires a metadata field...?

Reply:
[CUMULUS-3981](https://bugs.earthdata.nasa.gov/browse/CUMULUS-3981) has more information regarding the update on @smithy/smithy-client 4.1.0.
The actual ServiceException thrown from sdk does have $metadata field.